client/index.html

```diff
@@ -3,9 +3,9 @@
 	<head>
 		<script src="js/jquery.min.js"></script>
 		<script src="js/bootstrap.bundle.min.js"></script>
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.1.4/Chart.min.js"></script>      
```

Comment:
This works when our computers have internet. But won't work when the DAS is in the countryside of Victoria 😢. I'm happy to fix this for you though. Just letting you know!